src/commands/diff.ts

```diff
@@ -0,0 +1,109 @@
+import { flags } from '@oclif/command';
+import * as diff from '@asyncapi/diff';
+import * as parser from '@asyncapi/parser';
+import { load } from '../models/SpecificationFile';
+import Command from '../base';
+import { ValidationError } from '../errors/validation-error';
+import { SpecificationFileNotFound } from '../errors/specification-file';
+
+export default class Diff extends Command {
+  static description = 'find diff between two asyncapi files';
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    format: flags.string({
+      char: 'o',
```

Comment:
sounds strange to me that main command is `format` and short version is `o` and not `f`

Reply:
oops, yeah it should be `f`. Will fix that.